app/models/author.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Author < ApplicationRecord
```

Comment:
I have a hard time with this class name, because this is not a table of Authors in my mind. "Adam Wead" and "Ryan Schenk" (who are authors, without the capital-A) will show up multiple times in this table, once per WorkVersion or Collection that they are listed on. So in my mind this class does not model an Author, it models an author being attached to a particular thing. I like the "Authorship" name that you use on the relation, maybe that would be a better name for this class and table?